packages/react/src/components/ContainedList/ContainedList.mdx

```diff
@@ -100,6 +100,10 @@ render inline to the Contained List title, please see the
 also remain persistent under the title, so that it remains visible on scroll,
 when there are many list items passed in.
 
+`ContainedList.ContainedListItem` is deprecated, use
```

Comment:
could you please move this message below?  this is coming in between the persistent search elaboration